CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Default PriorityClasses for all components
```

Comment:
Note: Requires Kubernetes v1.14+ [stable]

Reply:
Should not be an issue, we are already only promise support for v1.20 +